routes/auth.route.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+var express = require('express');
+var router = express.Router();
+var mongoose = require('mongoose');
+
+require('../models/users.models');
+var User = mongoose.model('Users');
+
+
+
+module.exports = function(app, passport){
+   function linkedinSignInCallback(req, res, next) {
+    passport = req._passport.instance;
+    passport.authenticate('linkedin',function(err, user, token) {
```

Comment:
Indentation... Use 2 spaces for your indentation. One detected here and the line above